ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -49,7 +49,14 @@ const SYSV_CALL_STACK_ALIGN: usize = 16;
 /// On X86_64 the stack grows down.
 const STACK_DIRECTION: StackDirection = StackDirection::GrowsDown;
 
-extern "C" fn __yk_deopt(_ctrlpvars: *const c_void, _frameaddr: *const c_void) {
+extern "C" fn __yk_deopt(_ctrlpvars: *const c_void, _frameaddr: *const c_void, deoptid: usize) {
+    let ctr = crate::mt::THREAD_MTTHREAD
+        .with(|mtt| mtt.running_trace().unwrap())
+        .as_any()
+        .downcast::<X64CompiledTrace>()
+        .unwrap();
+    debug_assert!(deoptid < ctr.deoptinfo.len());
+    eprintln!("deopt frames: {:?}", ctr.deoptinfo[deoptid].frames);
```

Comment:
Debug?

Reply:
This is in lieu of ways to test this properly. I want to check that the deoptinfo is actually there and has the right content. Maybe the `debug_assert` is enough here?

Reply:
Yes, I think just the `debug_assert` is the right thing to do.

Reply:
I think he wants to match the output with fm until proper deopt is implemented?

Reply:
Ah I see!